snippets/messaging/cloud_messaging.py

```diff
@@ -220,3 +220,72 @@ def unsubscribe_from_topic():
     # for the contents of response.
     print(response.success_count, 'tokens were unsubscribed successfully')
     # [END unsubscribe]
+
+
+def send_all():
+    registration_token = 'YOUR_REGISTRATION_TOKEN'
+    # [START send_all]
+    # Create a list containing up to 100 messages.
+    messages = [
+        messaging.Message(
+            notification=messaging.Notification('Price drop', '5% off all electronics'),
```

Comment:
Those % signs in a string literal don't cause any problems, I guess?

Reply:
Not a problem in this context.